app/routes/groups_.$groupId.tsx

```diff
@@ -1,21 +1,66 @@
-import type { LoaderFunction } from '@remix-run/node';
+import type { Event } from '@prisma/client';
+import type { LoaderFunction, LoaderFunctionArgs, SerializeFrom } from '@remix-run/node';
 import { useParams, useLoaderData, Link } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
+import { EventCard } from '~/modules/events/card';
 import { json } from '@remix-run/node';
 
-export const loader: LoaderFunction = async ({ params }) => {
-  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
+export const loader: LoaderFunction = async ({ params }: LoaderFunctionArgs) => {
+  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId }, include: { events: true } });
   return json({ group });
 };
 
+export function useDeserializeGroup() {
```

Comment:
Let's move this to our hooks folder if possible! Would make it more reusable for other components.